Azure-Samples/AzureStorageSnippets
# Snippet project for quickstart-blobs-javascript-browser

## Purpose

Snippet project for https://docs.microsoft.com/en-us/azure/storage/blobs/quickstart-blobs-javascript-browser

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/AzureStorageSnippets.git
cd AzureStorageSnippets
git checkout master
npm install
```

* Test the code

See article https://docs.microsoft.com/en-us/azure/storage/blobs/quickstart-blobs-javascript-browser

## What to Check
Verify that the following are valid
See article https://docs.microsoft.com/en-us/azure/storage/blobs/quickstart-blobs-javascript-browser

## Other Information
